recipes/haploflow/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set version = "0.2" %}
+{% set sha256 = "a7428102197bde51246efb568c4351457a5e9f8ec08fbc8a63b4f6eb7fa77aff" %}
+
+package:
+  name: haploflow
+  version: {{ version }}
+
+source:
+  url: https://github.com/hzi-bifo/Haploflow/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+  patches: 
+    - bioconda-adjustments.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - cmake >=2.8
+  host:
+    - boost-cpp
+  run:
+    - boost-cpp
+
+
+test:
+  commands:
+    - haploflow --help
+
+about:
+  home: https://github.com/hzi-bifo/Haploflow
+  license: GPL-3.0 License
```

I've opened https://github.com/hzi-bifo/Haploflow/issues/8 to ask whether `GPL-3.0-only` or something else is the intended license here.

---

I got some clarifications from upstream:

> \[...\] I was waiting for the software to be published in a peer-reviewed journal to create a 1.0 release. Before that, v0.1 is probably the better version to use.

So, it's probably better to use the `v0.1` tag for now.
And
> \[...\] v0.1 used Apache and I updated the README.md when we changed to GPL for v0.2, but forgot to also update it to 0.2 in the text \[...\]

So, if you change this to use `v0.1`, then use
```suggestion
  license: Apache-2.0 AND MIT
```
here.